deflect/StreamSendWorker.h

```diff
@@ -62,20 +63,32 @@ class StreamSendWorker
     ~StreamSendWorker();
 
     /** Enqueue an image to be send during the execution of run(). */
-    Stream::Future enqueueImage(const ImageWrapper& image);
+    Stream::Future enqueueImage(const ImageWrapper& image, bool finish);
+
+    Stream::Future enqueueFinish(); //!< Enqueue a finishFrame()
```

Comment:
I don't understand what you mean here. It's standard doxygen, and we use it everywhere?